package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gitcoin",
+  "version": "1.0.0",
+  "description": "<img src='https://d3vv6lp55qjaqc.cloudfront.net/items/263e3q1M2Y2r3L1X3c2y/helmet.png'/>",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint app/static/v2/js/**/*.js",
+    "build": "webpack --config webpack.config.js --progress --colors",
+    "build-production": "webpack --config webpack.prod.config.js --progress --colors",
+    "watch": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ethikz/web.git"
+  },
+  "author": "Josh Christensen http://github.com/ethikz",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ethikz/web/issues"
+  },
+  "homepage": "https://github.com/ethikz/web#readme",
+  "devDependencies": {
+    "babel": "^6.23.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-env": "^1.6.1",
+    "eslint-friendly-formatter": "^3.0.0",
+    "eslint-loader": "^1.9.0",
```

Comment:
++ `eslint` ? folks might not have it globally installed
If we do that , we'll have to update the [lint script](https://github.com/gitcoinco/web/pull/111/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R8) to use the `eslint` found within `node_modules`.

Something like: `"lint": "./node_modules/../eslint app/static/v2/js/**/*.js"`,

Correct me if I'm wrong, I'd have to run `npm run lint` to run the linter right ?

I'm guessing we'll have to link this up to docker later , thus ensuring the docker image has everything you need
That way we can trigger the linter to kick on upon starting the application and on code being changed

What do you think ? 
I'm on two minds @owocki your thoughts ?

Reply:
i dont have any experience with this so i humbly defer to you :)

Reply:
Yea we will need to link it up in docker to automatically run the command.  I just wanted to get the config added for now because all of the files haven't been converted and linting will fail